diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 131352c3710075b69ab19aaf0508720a846d0916..1bfa2cde02a3585de4f6ddc79dd0249c363f229b 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -337,6 +337,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
