diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 3ed67d747dbebe93903268d8d8c1a1b60510e231..d9039989439a81260de9b8136cc44e66889c1fb3 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2761,6 +2761,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
