diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
index 5a51289c5219910d359a1f25a09269b0dc36ffea..df5776a940b2dc68ac3770cb930f4c9867d95e7b 100644
--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
+++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
@@ -1596,6 +1596,7 @@ const char* ChromeBrowsingDataRemoverDelegate::GetHistogramSuffix(
       return "WebrtcVideoPerfHistory";
     case TracingDataType::kMediaDeviceSalts:
       return "MediaDeviceSalts";
+    CHROME_BROWSING_DATA_REMOVER_DELEGATE_GET_HISTORY_SUFFIX
   }
 }
 
