diff --git a/chrome/browser/ui/views/tabs/tab_style_views.cc b/chrome/browser/ui/views/tabs/tab_style_views.cc
index e054a3a44ccd19167b9794f28dcd30705ad430eb..e03c472dd9af21b2c3437a37eb79de6dcf6ceafd 100644
--- a/chrome/browser/ui/views/tabs/tab_style_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_style_views.cc
@@ -135,6 +135,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   TabStyle::TabSelectionState GetSelectionState() const;
 
  private:
+  BRAVE_GM2_TAB_STYLE_H
   // Gets the bounds for the leading and trailing separators for a tab.
   TabStyle::SeparatorBounds GetSeparatorBounds(float scale) const;
 
@@ -354,6 +355,7 @@ SkPath TabStyleViewsImpl::GetPath(TabStyle::PathType path_type,
   const ShapeModifier shape_modifier = GetShapeModifier(path_type);
   const bool extend_left_to_bottom = shape_modifier & kNoLowerLeftArc;
   const bool extend_right_to_bottom = shape_modifier & kNoLowerRightArc;
+  extension_corner_radius = 0;
 
   SkPath path;
 
@@ -655,9 +657,10 @@ TabStyle::SeparatorBounds TabStyleViewsImpl::GetSeparatorBounds(
   TabStyle::SeparatorBounds separator_bounds;
 
   const int extra_vertical_space =
+      2 * (
       aligned_bounds.height() -
       (separator_size.height() + separator_margin.bottom() +
-       separator_margin.top());
+       separator_margin.top()));
 
   separator_bounds.leading = gfx::RectF(
       aligned_bounds.x() + corner_radius - separator_margin.right() -
@@ -1022,14 +1025,18 @@ void TabStyleViewsImpl::PaintSeparators(gfx::Canvas* canvas) const {
                                                    SK_AlphaOPAQUE));
   };
 
+  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
+  // will be approximated with color, although extremely subtle and
+  // likely only on screens >= 2x (i.e. separator width is 2+px)!
+  const int separator_radius =  separator_bounds.leading.width() / 2;
   cc::PaintFlags flags;
   flags.setAntiAlias(true);
   flags.setColor(separator_color(separator_opacities.left));
   canvas->DrawRoundRect(separator_bounds.leading,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
   flags.setColor(separator_color(separator_opacities.right));
   canvas->DrawRoundRect(separator_bounds.trailing,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
 }
 
 float TabStyleViewsImpl::GetTopCornerRadiusForWidth(int width) const {
