diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 46a9fac8b2716df2e3b6c62c7947308ff5425134..2b992c3db03bce2983e458222bc1564ba773edf9 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
