diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 6edbef34112ccae4e7e3e4f96ffacdb430234497..cbb3c96dcf3c598bb7932d2ac6c9f8f9b44395ae 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
