diff --git a/BUILD.gn b/BUILD.gn
index 3565821cfe79a545286ac072aaf898f436d3c7d7..06a9b2224abb3aef6580e5c06f22411cc50b61b2 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -746,6 +746,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3117,6 +3118,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
