diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 850089df41b3cb999acb88fc302971de494d8bfe..8b82e880f5de0d1719d8cf7dd22e8fb779480ef3 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3019,6 +3019,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
