diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index a028d1e75767c639e4e20dce38135094cf7e65dc..7cc28f818d012ca2a4e2f18168b73c8c38d52dc7 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3019,6 +3019,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
